protos/flyteidl/external/external_event.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package flyteidl.external;
+
+option go_package = "github.com/lyft/flyteidl/gen/pb-go/flyteidl/external";
+
+import "flyteidl/core/execution.proto";
+import "flyteidl/core/identifier.proto";
+import "flyteidl/core/catalog.proto";
+import "flyteidl/core/workflow_closure.proto";
+import "flyteidl/core/compiler.proto";
+import "flyteidl/admin/event.proto";
+import "flyteidl/admin/common.proto";
+import "flyteidl/admin/execution.proto";
+import "flyteidl/admin/node_execution.proto";
+import "flyteidl/admin/task_execution.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/struct.proto";
+
+message ExecutionEventClosure {
+    flyteidl.core.ConnectionSet connections = 1;
+
+    flyteidl.core.LiteralMap computed_input = 2;
+
+    core.Identifier launch_plan        = 3;
+    flyteidl.admin.Annotations annotations = 4;
+    // parent_node_execution
+    flyteidl.admin.ExecutionMetadata metadata = 5;
```

Comment:
This is static throughout the course of the execution, what's the decision to emit on every event?